binding/form_mapping.go

```diff
@@ -148,16 +150,22 @@ func setFloatField(val string, bitSize int, field reflect.Value) error {
 }
 
 func setTimeField(val string, structField reflect.StructField, value reflect.Value) error {
-	timeFormat := structField.Tag.Get("time_format")
-	if timeFormat == "" {
-		return errors.New("Blank time format")
-	}
 
 	if val == "" {
 		value.Set(reflect.ValueOf(time.Time{}))
 		return nil
 	}
 
+	timeFormat := structField.Tag.Get("time_format")
+	if timeFormat == "" {
+		t, err := dateparse.ParseAny(val)
```

Comment:
What location does `ParseAny` assume?

See below that the current behavior is using local timezone by default, unless a `utc` tag is informed in the field. I think it is desirable to keep this behavior.